packages/core/src/desmosclient.integration.spec.ts

```diff
@@ -257,8 +257,9 @@ describe("DesmosClient", () => {
       const saveProfileResult = await profileClient.signAndBroadcast(
         profileAddress,
         [msgSaveProfile],
-        "auto"
+        200000
```

Comment:
Using "auto" does not calculate the fees correctly, errors always happen since out of gas